components/admin/home/events-table.vue

```diff
@@ -21,15 +41,27 @@ const { data: events } = useEvents()
           {{ event.description }}
         </f7CardContent>
         <f7CardFooter>
-          <f7Button tonal>
+          <f7Button tonal @click="() => selectedEvent = event">
             Open
           </f7Button>
+          <f7Button tonal color="red" @click="() => selectedEvent = event" class="popup-open" data-popup=".delete-event-popup">
+            Delete
+          </f7Button>
         </f7CardFooter>
       </f7Card>
     </div>
 
     <div class="hidden md:inline">
       <div class="data-table data-table-init card">
+        <div class="card-header">
+          <div class="data-table-title">Events</div>
+          <div></div>
```

Comment:
Is this required?

Reply:
Wanted to make it a more defined section